ShellScript/Bash.sublime-syntax

```diff
@@ -1991,6 +1995,10 @@ contexts:
     - match: \\[$`"\\]
       scope: constant.character.escape.shell
 
+  string-path-separators:
+    - match: ':(?!$|\s|//)'
```

Comment:
```suggestion
    - match: ':(?=\$|/)'
```
Perhaps this would be a decent approximation for a list of allowed characters after a colon when combining multiple paths together? It should be very rare to combine two relative paths.

Reply:
Theoretical should also handle `Class::method()`. There may be various possible false positives anyway.

Reply:
Found another accidental match in `'([^.[:space:]]+)'`

Reply:
applied FichteFoll's suggestion, and added syntax tests to prove those examples are not scoped/treated as path separators

Reply:
May something like `/path/:~/dir` be something to considder? If yes, ... `:(?=\$|~?/(?!/))`

Reply:
And the current (or a relative) directory `/path:.:../sibling`.

Can't think of anything else right now.

Reply:
done

Reply:
Not sure why you require `.` and `~` to be followed by a slash, which is not a technical requirement, but it should be unlikely enough to not be relevant in practice, at least for `~`. `.` I could see but is probably also equivalent to an empty segment (which I remember seeing sometimes but probably way back on Windows).

Current changes are fine imo, but feel free to revise.

Reply:
You mean something like ...

```
- match: ':(?=\$|~|\.\.?|/(?!/))'
```

I haden't `bla:~:bla` in mind when writing my proposal.

Reply:
The current regex suits my personal use case, and I'm worried that making it more lenient could cause more false positives. My thinking would be to merge as is, and tweak it based on feedback later. But if we want, we can change it now also, I don't mind, just need someone to make the decision :)

Reply:
I made it for you ;-) as somehow sharing concerns about false positives.